scripts/helm/ort-server/Chart.lock

```diff
@@ -1,12 +1,12 @@
 dependencies:
 - name: keycloak
   repository: https://charts.bitnami.com/bitnami
-  version: 13.4.1
+  version: 24.4.13
```

Comment:
In Compose version 26.2.0 is used.

Reply:
It seems that in the meanwhile, the docker compose file was updated. I will adjust it.